### PR TITLE
Built transaction page - issue16

### PR DESCRIPTION
**Pull request by geniusFulldev**

### CODE REVIEW

**Review by abstract829 — CHANGES_REQUESTED** [review]
Looks great, 

The unique thing missing is that the UI is breaking below 1280px

We are not supporting full responsive but should at least works fine for +1024px 

Captura_de_pantalla_2023-11-24_a_las_15 22 26

Please fix it, with some scroll like this is fine, do not overthink it:

https://github.com/kwickbit/kwickbit_frontend_v0/assets/56497105/b76f431a-1ba7-443d-880c-704aa0d728e9



Notes:

Please avoid to hardcoded styles in tailwind like :  ```pt-[21px] pb-[18px]``` unless is a specific color you need, if you see in the design these values just use the closest one in tailwind

example: ```pt-[21px] pb-[18px]```  -> ```pt-5 pb-4```

**Comment by geniusFulldev** [comment]
Got it,  I will fix asap

**Comment by geniusFulldev** [comment]
Pushed the fixes

File: src/components/transactions/TransactionsView.tsx
Comment (abstract829) [inline src/components/transactions/TransactionsView.tsx:1]: Please change 

```bash
<div className="container-2xl mx-auto overflow-x-auto mt-12 px-4">
    <div className="w-fit">
        <TransactionSelectSection />
        <TransactionListSection
            transactions={data?.transactions}
            onRefresh={refetch}
        />
    </div>
</div>
   ```
   
   to 
```bash
   <div className="max-w-7xl px-4 mx-auto">
      <div className="overflow-x-auto">
        <div className="w-fit">
          <TransactionSelectSection />
          <TransactionListSection
            transactions={data?.transactions}
            onRefresh={refetch}
          />
        </div>
      </div>
    </div>
```
    
so the cards doesnt take more width than the header 
    
Captura de pantalla 2023-11-24 a la(s) 17 20 58

Captura de pantalla 2023-11-24 a la(s) 17 22 48

File: src/components/transactions/TransactionListSection.tsx
Comment (abstract829) [inline src/components/transactions/TransactionListSection.tsx:1]: replace ```shadow-[0_0_1px_0_rgba(23,26,31,0.07),0_0_2px_0_rgba(23,26,31,0.12)]```

with ```shadow border```

**Comment by abstract829** [comment]
@geniusFulldev Nice looks great now, i just mention some small changes in the styles and we are ready

**Comment by geniusFulldev** [comment]
@abstract829 Everything's ok?

**Review by abstract829 — CHANGES_REQUESTED** [review]

**Review by celadari — APPROVED** [review]
Great work ! :100: :100:

File: src/components/transactions/TransactionsView.tsx
Comment (geniusFulldev) [reply src/components/transactions/TransactionsView.tsx:1]: Will do

File: src/components/transactions/TransactionListSection.tsx
Comment (geniusFulldev) [reply src/components/transactions/TransactionListSection.tsx:1]: will do

**Comment by geniusFulldev** [comment]
Done

**Review by celadari — APPROVED** [review]

**Review by abstract829 — APPROVED** [review]

**Pull request merged**